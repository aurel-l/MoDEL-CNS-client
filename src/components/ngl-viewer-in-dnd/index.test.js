import React from 'react';
import { shallow } from 'enzyme';

import NGLViewerInDND from '.';

describe('<NGLViewerInDND />', () => {
  it('should render', () => {
    expect(
      shallow(<NGLViewerInDND accession="MCNS00001" analysis="fluctuation" />),
    ).toMatchSnapshot();
  });
});
