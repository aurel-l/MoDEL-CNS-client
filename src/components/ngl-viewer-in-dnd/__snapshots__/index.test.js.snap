// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NGLViewerInDND /> should render 1`] = `
<Rnd
  bounds="body"
  cancel="canvas, [data-popover]"
  className="rnd"
  data-rnd={true}
  maxHeight={9007199254740991}
  maxWidth={9007199254740991}
  onDrag={[Function]}
  onDragStart={[Function]}
  onDragStop={[Function]}
  onResize={[Function]}
  onResizeStart={[Function]}
  onResizeStop={[Function]}
  scale={1}
>
  <ForwardRef
    className="floating-card"
    elevation={4}
  >
    <Suspense
      fallback={null}
    >
      <lazy
        accession="MCNS00001"
        className="ngl-viewer-with-controls"
        close={[Function]}
        noTrajectory={false}
        requestedFrame={false}
        startsPlaying={false}
      />
    </Suspense>
  </ForwardRef>
</Rnd>
`;
